public/style/main.css

```diff
@@ -168,8 +326,11 @@ label {
 
 
 .isochroneSlider {
-  max-height : 33px;
-  overflow : hidden;
+  height : 30px;
+  width: 300px;
+  overflow : auto;
```

Comment:
why would be have auto overflow if you are dealing with fixed width columns - make sure you are designing for the internet - not just your browser on your screen